fetch_data.py

```diff
@@ -263,7 +265,11 @@ def summarize_within_poly(ds, poly, dim_encodings, varname="Gray", roundkey="Gra
 
 def geotiff_zonal_stats(poly, arr, transform, stat_list):
     poly_mask_arr = zonal_stats(
-        poly, arr, affine=transform, nodata=np.nan, stats=stat_list,
+        poly,
```

Comment:
Minor thing but what is causing changes like this? Is this `black`? I feel like I've seen formatting stuff like this going back and forth in various parts. It might just be line length. I think my `black` defaults to line lengths of 88, so this gets set back to the way it was originally if I were to run `black` on it.

Reply:
Yep, definitely a Black thing. This line of code was 69 characters long before the Black reformatting, so it doesn't seem to be a line length issue. I think Black maybe just doesn't like function parameters listed on their own line like this. I made sure I'm running the latest version of Black (22.1.0) with default settings, so I'll keep this change as-is for now. Not sure what's going on or why our version are acting differently.